scripts/zones/Aht_Urhgan_Whitegate/npcs/Lathuya.lua

```diff
@@ -9,37 +9,143 @@ require("scripts/globals/quests")
 require("scripts/globals/npc_util")
 -----------------------------------
 
+local craftingItems = {
+    [1] = -- magus bazubands
+    {
+        materials = {754, 828, 879, 4158},
+        currency = 2186,
+        currencyAmt = 2,
+        result = 14928
+    },
+    [2] = -- magus shalwar
+    {
+        materials = {761, 828, 2175, 2340},
+        currency = 2186,
+        currencyAmt = 2,
+        result = 15600
+    },
+    [3] = -- magus jubbah
+    {
+        materials = {828, 2229, 2288, 2340},
+        currency = 2186,
+        currencyAmt = 4,
+        result = 14521
+    }  
+}
+
 function onTrade(player, npc, trade)
+    local remainingBLUAF = player:getCharVar("[BLUAF]Remaining") -- Bitmask of AF the player has NOT crafted
+    if remainingBLUAF >= 1 then
+        local currentTask = player:getCharVar("LathuyaCurrentTask_Mask")
```

Comment:
Is this being used anywhere?

Reply:
Nope, it _was_ but I guess I accidentally left it in there when reworking it; will remove. Nice eye~!